evm.md

```diff
@@ -1857,6 +1880,48 @@ In the YellowPaper, each opcode is defined to consume zero gas unless specified
     rule [#memoryUsageUpdate.some]: #memoryUsageUpdate(MU, START, WIDTH) => maxInt(MU, (START +Int WIDTH) up/Int 32) requires WIDTH  >Int 0
 ```
 
+Access List Gas
+---------------
+
+```k
+    syntax Bool ::= #usesAccessList ( OpCode ) [function]
+ // -----------------------------------------------------
+    rule #usesAccessList(OP) => true requires OP ==K BALANCE
+                                       orBool OP ==K SELFDESTRUCT
+                                       orBool OP ==K EXTCODEHASH
+                                       orBool OP ==K EXTCODESIZE
+                                       orBool OP ==K EXTCODECOPY
+                                       orBool OP ==K SLOAD
+                                       orBool OP ==K SSTORE
+                                       orBool OP ==K CALL
+                                       orBool OP ==K CALLCODE
+                                       orBool OP ==K DELEGATECALL
+                                       orBool OP ==K STATICCALL
+    rule #usesAccessList(_) => false [owise]
```

Comment:
It's unlikely this will work well with the Java backend (the `owise`). So you may need to just say that `#usesAccessList` is `functional`, and have it go directly to the large requires clause you've written out.